{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/globals.py

```diff
@@ -0,0 +1,144 @@
+####################
+# Project metadata #
+####################
+
+PROJECT_NAME = '{{cookiecutter.package_name}}'
+CLUSTER_PROJECT = 'proj_cost_effect'
+
+CLUSTER_QUEUE = 'all.q'
+MAKE_ARTIFACT_MEM = '3G'
+MAKE_ARTIFACT_CPU = '1'
+MAKE_ARTIFACT_RUNTIME = '3:00:00'
+
+LOCATIONS = [
+    # TODO - project locations here
+]
+
+
+#############
+# Data Keys #
+#############
+
+METADATA_LOCATIONS = 'metadata.locations'
+
+POPULATION_STRUCTURE = 'population.structure'
+POPULATION_AGE_BINS = 'population.age_bins'
+POPULATION_DEMOGRAPHY = 'population.demographic_dimensions'
+
+ALL_CAUSE_CSMR = 'cause.all_causes.cause_specific_mortality_rate'
+
+# TODO - sample disease keys
+DIARRHEA_CAUSE_SPECIFIC_MORTALITY_RATE = 'cause.diarrheal_diseases.cause_specific_mortality_rate'
+DIARRHEA_PREVALENCE = 'cause.diarrheal_diseases.prevalence'
+DIARRHEA_INCIDENCE_RATE = 'cause.diarrheal_diseases.incidence_rate'
+DIARRHEA_REMISSION_RATE = 'cause.diarrheal_diseases.remission_rate'
+DIARRHEA_EXCESS_MORTALITY_RATE = 'cause.diarrheal_diseases.excess_mortality_rate'
+DIARRHEA_DISABILITY_WEIGHT = 'cause.diarrheal_diseases.disability_weight'
+DIARRHEA_RESTRICTIONS = 'cause.diarrheal_diseases.restrictions'
+
+
+### ------------- Reviewers: what, if any, of the following should be part of the template ------------- 
+'''
+###########################
+# Disease Model variables #
+###########################
+
+# TODO - sample states and transitions
+DIARRHEA_MODEL_NAME = 'diarrheal_diseases'
+DIARRHEA_SUSCEPTIBLE_STATE_NAME = f'susceptible_to_{DIARRHEA_MODEL_NAME}'
+DIARRHEA_WITH_CONDITION_STATE_NAME = DIARRHEA_MODEL_NAME
+DIARRHEA_MODEL_STATES = (DIARRHEA_SUSCEPTIBLE_STATE_NAME, DIARRHEA_WITH_CONDITION_STATE_NAME)
+DIARRHEA_MODEL_TRANSITIONS = (
+    f'{DIARRHEA_SUSCEPTIBLE_STATE_NAME}_TO_{DIARRHEA_WITH_CONDITION_STATE_NAME}',
+    f'{DIARRHEA_WITH_CONDITION_STATE_NAME}_TO_{DIARRHEA_SUSCEPTIBLE_STATE_NAME}',
+)
+
+# TODO - add all diseases
+DISEASE_MODELS = (DIARRHEA_MODEL_NAME)
```

Comment:
This is pretty much the whole point of the section.  If we could extract this info from somewhere else and fill it here, that would be ideal.

Reply:
Some variation of the following might be worthwhile:

class AnnotateEntity:
    def __init__(self, the_type: str, name: str):
        self._type = the_type
        self._name = name
        self._state_susceptible = f'susceptible_to_{self._name}'
        self._state_with_condition = f'with_{self._name}'
        self._states = [
            self._state_susceptible,
            self._state_with_condition,
        ]
        self._transitions = (
            f'{self._state_susceptible}_TO_{self._state_with_condition}',
            f'{self._state_with_condition}_TO_{self._state_susceptible}',
        )
        
    @property
    def name(self) -> str:
        return self._name
        
    @property
    def model_map(self):
        return {
            'states': self._states,
            'transitions': self._transitions,
        }

DIARRHEA = AnnotateEntity('cause', 'diarrheal_diseases') 
MEASLES = AnnotateEntity('cause', 'measles')

MODEL_MAP = {
    DIARRHEA.name: DIARRHEA.model_map,
    MEASLES.name: MEASLES.model_map
}

Reply:
Skip the annotated entity for now